README.md

```diff
@@ -1,35 +1,58 @@
 # AutoTrain
 
-This AutoTrain competition has the goal to fully automatize the training
-process of deep learning models. Submitted optimizers will be benchmarked on a set of
-unknown architecture/dataset pairs. The winning optimizers will be made publicly available as
+AutoTrain challenges you to submit optimizers that work reliably on any deep learning task without task-specific tuning.
+It separates AutoML into (1) fully automatic training of a model and (2) model selection, and tackles the first aspect.
+
+Your submissions will be benchmarked on a secret set of architecture/dataset pairs inspired by common deep learning tasks.
+The optimizers need to achieve a target test loss as fast as possible. The fastest on average wins the competition.
+
+The winning optimizers will be made publicly available as
 open source and bring significant value to practitioners and researchers, by removing
 the need of expensive hyperparameter tuning, and by providing fair benchmarking of
 all optimizers.
 
-### Logistics & Timeline
+## Timeline
+
+| Date                |                                       |
+|---------------------|---------------------------------------|
+| October 1, 2019                  | Submission window opens  |
```

Comment:
let's clarify from when people can start preparing their submissions with our provided package (stable submission interface)

Reply:
Good one. Hmm. I think we should check if the current interface is okay for language-type tasks.
Otherwise, I think it should work. Shall we reserve 2 weeks for this checking and one meeting once more with a couple of people to talk it through?